src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Envelope.java

```diff
@@ -46,8 +49,7 @@
     private Timestamp zipFileCreatedDate;
     @JsonProperty("zip_file_name")
     private String zipFileName;
-    @JsonProperty("last_event")
-    private Event lastEvent = Event.ENVELOPE_CREATED;
+    private Event status = Event.ENVELOPE_CREATED;
```

Comment:
Should status should marked as JsonIgnore as we do not want to send this back to services ?